### ISSUE TITLE
NTR: 3-hydroxysebacic acid high in urine (Lab: chemistry: organic acids)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8618]
New term request 3-hydroxysebacic acid high in urine (segal_030513193512):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6001001
- label: Elevated urine 3-hydroxysebacic acid level
- parents: [Elevated urinary carboxylic acid (HP:0040156)](https://hpo.jax.org/app/browse/term/HP:0040156)
- pmid: 
- def: The amount of 3-hydroxysebacic acid in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8618
- synonyms:
